src/main/java/uk/ac/ox/oxfish/biology/growers/CommonLogisticGrowerFactory.java

```diff
@@ -15,9 +15,6 @@
      */
     private DoubleParameter distributionalWeight = new FixedDoubleParameter(-1);
 
-    private boolean usesBeginningOfYearBiomassToGrow = false;
```

Comment:
also removing this, I don't understand.  Is there a reason behind forcing growth to happen only after mortality rather than before to keep it comparable to the surplus model? Feels like this was a valid point made by Katyana

Reply:
The new `FadAwareLogisticGrower` still has the option to use last year's biomass. I removed it from the basic grower to keep it simple.

I think that the `FadAwareLogisticGrower` makes `CommonLogisticGrower` obsolete, as the former should also when there are no FADs, but I hesitated to swap it in completely until it is battle tested a bit.